examples/xgboost_simple.py

```diff
@@ -38,17 +38,20 @@ def objective(trial):
 
     param = {
         "silent": 1,
+        "seed": 1,
         "objective": "binary:logistic",
-        "booster": trial.suggest_categorical("booster", ["gbtree", "gblinear", "dart"]),
+        "booster": trial.suggest_categorical("booster", ["gbtree", "dart"]),
+        "min_child_weight": trial.suggest_loguniform("min_child_weight", 1e-3, 1e3),
+        "subsample": trial.suggest_uniform("subsample", 0.5, 1.0),
+        "colsample_bytree": trial.suggest_uniform("colsample_bytree", 0.5, 1.0),
         "lambda": trial.suggest_float("lambda", 1e-8, 1.0, log=True),
         "alpha": trial.suggest_float("alpha", 1e-8, 1.0, log=True),
+        "max_depth": trial.suggest_int("max_depth", 3, 10),
```

Comment:
In [reference](https://www.analyticsvidhya.com/blog/2016/03/complete-guide-parameter-tuning-xgboost-with-codes-python/), it is tuned in `{3, 5, 7, 9}`. How about setting `step = 2`?
```suggestion
        "max_depth": trial.suggest_int("max_depth", 3, 10, step=2),
```